server/main.js

```diff
@@ -0,0 +1,17 @@
+const app = require('./index');
+const PORT = 3000;
+const { db } = require('./db');
+
+const init = async () => {
+  await db.sync({ force: true }).then(() => {
```

Comment:
I'd be careful about keeping force:true in here since it'll wipe out your database every time the server starts up - this might cause issues later on. In general, I'd keep force:true limited to just being inside of your seed.

Reply:
good to know

Reply:
good to know.